pkg/api/v1alpha1/oidcconfig_webhook.go

```diff
@@ -29,7 +29,15 @@ var _ webhook.Validator = &OIDCConfig{}
 func (r *OIDCConfig) ValidateCreate() error {
 	oidcconfiglog.Info("validate create", "name", r.Name)
 
-	return nil
+	var allErrs field.ErrorList
+
+	allErrs = append(allErrs, r.Validate()...)
```

Comment:
do we need this? or can we do
```suggestion
	allErrs := r.Validate()
```